### PR TITLE
Add miscellaneous minor CLI features

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,9 @@
 * Automatic discovery of branch relations (`git machete discover`)
 
 
+A port to an IntelliJ plugin is under development, see [VirtusLab/git-machete-intellij-plugin](http://github.com/VirtusLab/git-machete-intellij-plugin/).
```
Comment (mkondratek) [inline README.md:24]: :heart:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,25 +12,26 @@ To run tests, execute `tox`.
 
 ## Command properties/classification
 
+Deprecated commands are excluded.
+
+Any command that can display status can also run `machete-status-branch` hook.
```
Comment (mkondratek) [inline CONTRIBUTING.md]: `the status`?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -12,25 +12,26 @@ To run tests, execute `tox`.
...
-Any command that can run rebase can also run `machete-pre-rebase` hook.
+| can run rebase                                                    | `reapply` (\*), `slide-out`, `traverse`, `update`                             |
+| expects no ongoing rebase/merge/cherry-pick/revert/am             | `go`, `reapply`, `slide-out`, `traverse`, `update`                            |
+| has stable output format across minor versions (plumbing command) | `file`, `fork-point` (\*\*), `is-managed`, `list`, `show`, `version`          |
 
-`reapply` can run rebase but can't run merge since merging a branch with its own fork point can always be performed in fast-forward manner and generally doesn't make much sense.
+(\*) `reapply` can run rebase but can't run merge since merging a branch with its own fork point is a no-op and generally doesn't make much sense.
 
-Stable output is only guaranteed for `fork-point` when invoked without options or with `--inferred` option.
+(\*\*) Stable output is only guaranteed for `fork-point` when invoked without options or with `--inferred` option.
```
Comment (mkondratek) [inline CONTRIBUTING.md]: `The stable output`?

Comment (PawelLipski) [reply CONTRIBUTING.md]: Changed to `A stable output` actually :)

**Pull request closed without merging**